lib/auth0/api/v2/actions.rb

```diff
@@ -16,10 +16,7 @@ module Actions
         # @param page [integer] The page number. Zero based.
         # @param installed [boolean] When true, return only installed actions. When false, return only custom actions. Returns all actions by default.
         # @return [json] Actions and pagination info
-        def actions(trigger_id, action_name, deployed: nil, per_page: nil, page: nil, installed: nil)
-          raise Auth0::MissingTriggerId, 'Must supply a valid trigger_id' if trigger_id.to_s.empty?
-          raise Auth0::MissingActionName, 'Must supply a valid action_name' if action_name.to_s.empty?
-
+        def actions(trigger_id: nil, action_name: nil, deployed: nil, per_page: nil, page: nil, installed: nil)
```

Comment:
I agree that this is the ideal API for this method but I would prefer we implement it in a non-breaking way. How about we do optional position parameters instead of named ones for the first two? That way existing calls will continue to work while there is also the ability to optionally leave them out.

```ruby
def actions(trigger_id = nil, action_name = nil, deployed: nil, per_page: nil, page: nil, installed: nil)
```